extension/public/popup.css

```diff
@@ -241,12 +242,14 @@ header {
 
   border-radius: 12px;
   background-color: #c54ccf;
+
```

Comment:
Please remove extra empty lines as it is not related to this commit. If there is a convention that you would like to follow, we can discuss it in a meeting and create a code formatting issue.